DSharpPlus/Entities/Channel/DiscordChannel.cs

```diff
@@ -179,6 +186,20 @@ public IEnumerable<DiscordChannel> Children
             }
         }
 
+        /// <summary>
+        /// Gets this channel's threads. This applies only to text and news channels.
+        /// </summary>
+        [JsonIgnore]
+        public IEnumerable<DiscordThreadChannel> Threads
```

Comment:
This should return `IReadOnlyList` not `IEnumerable`

Reply:
Corrected.